BatchForSendingResponseHandler.cls

```diff
@@ -0,0 +1,45 @@
+/**
+* @description: This is the handler for BatchForSendingResponse.In this all the
+                contacts are fetched from Contact_Email_Handler__c where IsResponded
+                field is not checked to true and response is sent back to them and 
+                Isresponded field is checked to true.
+* @author: Noopur Naik
+*
+* @revision:
+* Version      Date            Description
+* ------------------------------------------------
+* 1.0          30-Jun-2017     Initial revision.
+**/
+
+public with sharing class BatchForSendingResponseHandler {
+     
+     public static void recordsBasedOnResponse(List<Email_Handler__c> emailhandlerList)
+     {
+        Id templateId = [Select id from EmailTemplate where name=:'SearchEngineTemplateNew'].id;
```

Comment:
Give proper name to email template. Also this query can throw exception so need to handle proper null pointer exception here.